application/event_processor/event_processor.py

```diff
@@ -220,6 +245,7 @@ def lambda_handler(event: Dict[str, Any], context: LambdaContext) -> None:
     matching_services = event_processor.get_matching_services()
 
     if len(matching_services) == 0:
+        log_unmatched_nhsuk_pharmacies(nhs_entity)
         logger.error(f"No matching DOS services found that fit all " f"criteria for ODSCode '{nhs_entity.ODSCode}'")
```

Comment:
Is there any benefit to this line as you have a replacement in the function below - this line `No matching DOS services found in DOS for a given NHS pharmacy service`
``` 
logger.info(
        "No matching DOS services found in DOS for a given NHS pharmacy service",
        extra={
            "log_id": UN_MATCHED_PHARMACY_LOG_ID,
            "nhsuk_odscode": nhs_entity.ODSCode,
            "nhsuk_org_name": nhs_entity.OrganisationName,
            "nhsuk_service_type": nhs_entity.ServiceType,
            "nhsuk_service_status": nhs_entity.OrganisationStatus,
            "nhsuk_service_sub_sector": nhs_entity.OrganisationSubType,
            "nhsuk_address1": nhs_entity.Address1 if hasattr(nhs_entity, "Address1") else "",
            "nhsuk_address2": nhs_entity.Address2 if hasattr(nhs_entity, "Address2") else "",
            "nhsuk_address3": nhs_entity.Address3 if hasattr(nhs_entity, "Address3") else "",
            "nhsuk_address4": nhs_entity.Address4 if hasattr(nhs_entity, "Address4") else "",
            "nhsuk_address5": nhs_entity.Address5 if hasattr(nhs_entity, "Address5") else "",
            "nhsuk_postcode": nhs_entity.Postcode,
        },
    )
```

Reply:
Yes, kind of duplicating the same message.  I will remove "No matching DOS services  ...."